dictionaries/en_gb.dictionary.itop.core.php

```diff
@@ -0,0 +1,1292 @@
+<?php
+/**
+ * Localized data
+ *
+ * @copyright Copyright (C) 2010-2024 Combodo SAS
```

Comment:
The file is just created, so you can't claim copyright as of 2010.
```suggestion
 * @copyright Copyright (C) 2024 Combodo SAS
```

(The same for all files)

Reply:
true :-(
I check this

Reply:
Done (see 41082639)